### DISCUSSION TITLE
Framing Overlay adoption

### DISCUSSION THREAD

**Discussion by carlyh-micb** [the-human-colossus-foundation/oca-spec#43] [General]
The DSWG has proposed three framing overlays and described them in an RFC.
https://github.com/the-human-colossus-foundation/oca-spec/blob/master/RFCs/0004-framingOverlay.md

I would like to start a discussion on the RFC and the possible timeline for adoption into the next version of the OCA specification.

**Comment by blelump** [comment]
@carlyh-micb @pknowl, nothing stops us from going forward with it and releasing OCA v1.1. Prepare a formal pull request extending the spec with the proposed overlays.

**Reply by carlyh-micb** [reply]
Just checking, by preparing a formal pull request you mean for editing this: https://github.com/the-human-colossus-foundation/oca-spec/blob/master/docs/v1.1.0-rc.md (as in, we write the section for framing in this document?)

Or is the RFC where you want it?

**Reply by blelump** [reply]
I mean to extend the spec with the proposed changes in RFC and make a PR :-) . We're now adopting the RFC. Consult with @pknowl where to put it in the spec. By spec, I mean [this](https://github.com/the-human-colossus-foundation/oca-spec/blob/master/docs/specification/README.md)

**Comment by carlyh-micb** [comment]
![image](https://github.com/user-attachments/assets/3b383747-d825-47b1-9892-64bbf3584085)

What is the difference between the readme and the v1.1.0 document?

**Reply by blelump** [reply]
It was an old attempt to address problems that today we can say is a rabbit hole. Therefore, please ignore it.

**Comment by carlyh-micb** [comment]
https://github.com/the-human-colossus-foundation/oca-spec/blob/Feat-framing_overlays/docs/specification/README.md#framing-overlays

On this branch we have made adjustments to the specification to add framing.

We are looking for feedback to have it officially added to the specification.

**Reply by blelump** [reply]
@carlyh-micb, please create a PR so we can review it.

**Reply by carlyh-micb** [reply]
A new PR has been done: https://github.com/the-human-colossus-foundation/oca-spec/pull/64

**Reply by blelump** [reply]
@carlyh-micb can you add changes from #64 into #63?

**Reply by carlyh-micb** [reply]
No I can't, I don't know how, it is beyond my GitHub abilities. I tried and failed and it was really screwing up so I created a new PR.

**Comment by mitfik** [comment]
I would like to raise a question in regards of `framing overlay` how it correspond to the `classification` attribute from `capture based` and why we need it if we can frame given object to any specific "industry" through that mechanism?

What would be role of `classification` after introducing framing overlay?

**Comment by pknowl** [comment]
**The classification attribute and framing overlays serve different but complementary roles.**

The **classification attribute** categorises a schema into a specific industry or sector using a standardised taxonomy (like GICS or SDGs). This categorisation is helpful for broad grouping and searching schemas based on industry or function, making it easier to find relevant schemas within a category.

**Framing overlays**, on the other hand, play a crucial role in providing semantic context by mapping specific schema elements (like attributes, units, or entry codes) to external vocabularies or ontologies. This framing is essential in ensuring interoperability and understanding across different systems.

However, it's important to note that the effectiveness of these tools can be influenced by the quality and consistency of the underlying data. Inaccurate or inconsistent data can lead to misclassification or misinterpretation, affecting the efficiency of schema management and interoperability.

In essence:

**Taxonomies** provide the **structure** for organising data into functional categories, making it easy to find and store.

**Ontologies** provide the **knowledge** and **context**, helping systems and users understand the meaning and relationships of that structured data.

**Framing overlays** provide the **bridge** between these two worlds.

Both taxonomies and ontologies are essential: taxonomies organise data, while ontologies help interpret and apply it meaningfully across different systems and domains. The importance of both cannot be overstated in terms of data management.

**Reply by mitfik** [reply]
> The classification attribute categorises a schema into a specific industry or sector using a standardised taxonomy (like GICS or SDGs). This categorisation is helpful for broad grouping and searching schemas based on industry or function, making it easier to find relevant schemas within a category.

If entity `A` would create schema in industry `iA` and someone would find it use full for his industry (same schema) he would need to recreate capture base to remove classification to be able to mark it with his industry `iB`. If the classification would be "enhanced" process this could potentially increase interoperability of the schema. Except very specific scheme a lot of schemas are cross industry e.g. person schema. It could be employee, patient, engineer etc... 

> Framing overlays

as I understood it could happen that I could have `capture base` with classification in `automotive industry` and framing overlay pointing to healthcare ontology. Since classification cannot be easily swapped does not generate that a problem ?
